translations/terminal_tutorial/terminal_tutorial.kor.md

```diff
@@ -0,0 +1,375 @@
+# [Contribute To This Project](https://syknapse.github.io/Contribute-To-This-Project/)
+
+![image info](/favicon.png)
+
+> :sparkling_heart: 으로 제작된 로고, 제작자: [CandidDeer](https://github.com/CandidDeer)
+
+[![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twit]
+
+[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This-Project/)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Contribute-To-This-Project/)
+
+---
+
+> ## **공지:**
+>
+> 이 프로젝트의 메인테이너가 되어 도움을 주고 싶으신가요? 관심이 있으시다면 메인테이너 가이드 [maintainer's guide](/maintainer_guide.md) 를 읽고 [Twitter](https://twitter.com/Syknapse) 에서 DM을 보내주세요.
```

Comment:
sadly we don't have any korean translation for maintainer guide. if you happend to be do one, don't forget to change this link.

Reply:
I'll try to add the Korean translation for maintainer's guide when I get a chance. Thanks!